README.md

```diff
@@ -101,14 +101,409 @@ download or scan any previously scanned sources, the Scanner can be configured (
 e.g. on [Artifactory](./scanner/src/main/kotlin/ArtifactoryCache.kt) or S3 (not yet implemented). Using the example of
 configuring an Artifactory cache, the YAML-based configuration file would look like:
 
-```
+```yaml
 scanner:
   cache:
     type: Artifactory
     url: "https://artifactory.domain.com/artifactory/generic-repository-name"
     apiToken: $ARTIFACTORY_API_KEY
 ```
 
+## Tutorial
+
+This tutorial gives a brief introduction how the tools work together using the NPM package
+[mime-types](https://www.npmjs.com/package/mime-types) as example. It will lead through the main steps in running ORT:
+
+* Install ORT.
+* Analyze the dependencies of `mime-types` using the `analyzer`.
+* Scan the source code of `mime-types` and its dependencies using the `scanner`. 
+
+**1. Prerequisites**
+
+ORT is compatible with Linux, Mac, and Windows. This tutorial assumes that you are running on Linux, but it should be
+easy to adapt the commands to Windows.
+
+To run ORT the following tools are required:
+
+* Java (version >= 8)
+* Git
+
+For some of the supported package managers and SCMs additional tools need to be installed: 
+
+* CVS
+* Mercurial
+* NPM (required for this tutorial because we scan `mime-types` which is an NPM project)
+* SBT
+* Subversion
+* Yarn
+
+**2. Download & Install ORT**
+
+In future we will provide binaries of the ORT tools, but currently you have to build the tools on your own. First
+download the source code from GitHub (replace `[ort-path]` with the path you want to clone ORT to):
```

Comment:
Explicitly specifying a path to clone to is quite uncommon, IMO. Should we simply omit this and assume the clone path to be the default of `oss-review-toolkit`?